addon/utils/regexp.js

```diff
@@ -0,0 +1,7 @@
+const versionRegExp = /\d[.]\d[.]\d/;
+const shaRegExp = /[a-z\d]{8}/;
+
+export default {
```

Comment:
Gotcha. If you ever wanted them both as one object, you could also do

```js
import * as regExpObject from 'ember-cli-app-version/utils/regexp';
```

too.